docs/architecture.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Architecture
+weight: 20
+description: >
+  Architecture of Seccomp Agent.
+---
+
+
+## From Kubernetes to Seccomp Agent
+
+![From Kubernetes to Seccomp Agent](kubernetes-to-seccomp-agent.png "From Kubernetes to Seccomp Agent")
+
+Steps:
```

Comment:
I'd mention that in this example we use containerd, but this _can_ be run with other container runtimes (docker supports it now, for example).